src/plugins/vis_type_vislib/public/pie_fn.ts

```diff
@@ -59,10 +59,14 @@ export const createPieVisFn = (): VisTypeVislibPieExpressionFunctionDefinition =
       help: 'vislib pie vis config',
     },
   },
-  fn(input, args) {
+  fn(input, args, handlers) {
     const visConfig = JSON.parse(args.visConfig) as PieVisParams;
     const visData = vislibSlicesResponseHandler(input, visConfig.dimensions);
 
+    if (handlers?.inspectorAdapters?.tables) {
```

Comment:
cc @nickofthyme Just want to make you aware of this change - it's probably necessary to do it in the vis_type_xy as well

Reply:
this is a weird one .... actually this should always be there, but in tests we are not passing it in